helm_deploy/values-preprod.yaml

```diff
@@ -29,3 +28,7 @@ generic-prometheus-alerts:
     - "hmpps-move-and-improve-preprod-keyworker_domain_events_dlq"
   sqsAlertsTotalMessagesThreshold: 1
   businessHoursOnly: true
+  applicationCronJobStatusFailedWindowMinutes: 15
```

Comment:
Guessing this is what prometheus is alerting us to. What's the default for this?